e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -49,34 +55,46 @@ context('When fee record correction feature flag is enabled', () => {
   });
 
   context('PDC_RECONCILE users can create record correction requests', () => {
+    const { createFeeRecordCorrectionRequestPage } = pages;
+
     beforeEach(() => {
       pages.landingPage.visit();
       cy.login(USERS.PDC_RECONCILE);
 
       cy.visit(`utilisation-reports/${reportId}`);
     });
 
-    it('should display error message if invalid selections for creating fee record correction request', () => {
+    it('should be able to create a record correction request', () => {
+      premiumPaymentsTab.premiumPaymentsTable.checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status).click();
+
       premiumPaymentsTab.createRecordCorrectionRequestButton().should('exist');
       premiumPaymentsTab.createRecordCorrectionRequestButton().click();
 
-      cy.url().should('eq', relative(`/utilisation-reports/${reportId}`));
+      cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}`));
 
-      errorSummary().contains('Select a record to create a record correction request');
-      premiumPaymentsTab.premiumPaymentsTable.error().should('exist');
-      cy.assertText(premiumPaymentsTab.premiumPaymentsTable.error(), 'Error: Select a record to create a record correction request');
-    });
+      //---------------------------------------------------------------
+      // Create record correction request page
+      //---------------------------------------------------------------
+      cy.assertText(mainHeading(), 'Record correction request');
 
-    it('should be able to initiate record correction request', () => {
-      premiumPaymentsTab.premiumPaymentsTable.checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status).click();
+      feeRecordSummary.container().should('exist');
+      cy.assertText(feeRecordSummary.facilityId(), feeRecordAtToDoStatus.facilityId);
+      cy.assertText(feeRecordSummary.exporter(), feeRecordAtToDoStatus.exporter);
+      cy.assertText(feeRecordSummary.requestedBy(), `${USERS.PDC_RECONCILE.firstName} ${USERS.PDC_RECONCILE.lastName}`);
 
-      premiumPaymentsTab.createRecordCorrectionRequestButton().should('exist');
-      premiumPaymentsTab.createRecordCorrectionRequestButton().click();
+      createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT).check();
+      createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.OTHER).check();
+      cy.keyboardInput(createFeeRecordCorrectionRequestPage.additionalInfoInput(), 'Some additional info.');
 
-      cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}`));
+      cy.clickContinueButton();
+
+      //---------------------------------------------------------------
+      // Create record correction request - "check the information" page
```

Comment:
I don't think we need this comment it's just asserting a URL

Reply:
Ideally should be in it's own test i.e

```js
it('should redirect to ...')
```

But appreciate this wasn't created in this PR so not a blocker